testdata/prom.libsonnet

```
@@ -0,0 +1,46 @@
+// see https://github.com/grafana/jsonnet-libs/promsonnet
+{
+  v1: {
+    ruleGroupSet: {
+      new():: {
+        groups_map:: {},
+        groups_order:: [],
+        local groups_map = self.groups_map,
+        local groups_order = self.groups_order,
+        groups: [groups_map[group] for group in groups_order],
+      },
+      addGroup(group):: {
+        groups_map+:: {
+          [group.name]: group,
+        },
+        groups_order+:: [group.name],
+      },
+    },
+
+    ruleGroup: {
+      new(name):: {
```

```suggestion
    newRuleGroup(name):: {
```
Also, no need to next this.